rlclientlib/generic_event.cc

```diff
@@ -45,4 +47,14 @@ namespace reinforcement_learning {
   const generic_event::payload_buffer_t& generic_event::get_payload() const {
     return _payload;
   }
+
+  generic_event::encoding_type_t generic_event::get_encoding() const {
+    switch (_content_type) {
+      case event_content_type::ZSTD:
+        return encoding_type_t::EventEncoding_Zstd;
+      case event_content_type::IDENTITY:
+      default:
```

Comment:
seems leftover

Reply:
That's not a leftover, we need to handle default is some way otherwise it's UB.